tests/node/run_node_test.mjs

```diff
@@ -181,4 +181,11 @@ suite('Test Node.js', function () {
 
     assert.deepEqual(jsonAfter, json);
   });
+  test('Dropdown getText works with no HTMLElement defined', function () {
```

Comment:
Just to check: this test fails without the fix in place, right?

Reply:
Yes.